src/representations/calculator_spherical_invariants.hh

```diff
@@ -539,9 +537,10 @@ namespace rascal {
         soap_vector.normalize();
       }
       if (this->compute_gradients) {
+        auto ii_pair = center.get_atom_ii();
```

Comment:
I just want to remark that get_pair_ii would make more sense fore me, but nothing important to change